docs/developer-guide/mapstore-migration-guide.md

```diff
@@ -22,6 +22,14 @@ This is a list of things to check if you want to update from a previous version
 
 ## Migration from 2024.02.00 to 2025.01.00
 
+### HomeDescription plugin configuration changes
+
+The HomeDescription plugin has been refactored and a property has been removed:
+
+- `cfg.name` removed in favor of the translation message `home.shortDescription`
+
+For downstream projects review the `.less` or `.css` files for possible conflicting style related to the `.ms-home-description` class selector
```

Comment:
need also to indicate how to insert the favicon. Having a default to avoid migration and doumented how to customize could be good, but I don't know how feasible without a refactor.